packages/utilities/src/index.ts

```diff
@@ -1,9 +1,7 @@
-import { BusinessHour, isOutsideBusinessHours } from './is-outside-business-hours';
+import { isOutsideBusinessHours } from './is-outside-business-hours';
+import { PlaceholdersMap, BusinessHour } from './models/interfaces';
```

Comment:
I can see that you're importing BusinessHour, but don't see the usage of it

Reply:
It is imported and used in rule-engine/utils.ts